R/randomForest_tidiers.R

```diff
@@ -1008,6 +1017,15 @@ evaluate_classification <- function(actual, predicted, class, pretty.name = FALS
   ret
 }
 
+# with binary probability prediction model from ranger, take the level with bigger probability as the predicted value.
+get_binary_predicted_value_from_probability <- function(x) {
```

Comment:
It would be wonderful if you can add a testcase for this function.

Reply:
Added a test.